test.js

```diff
@@ -5,8 +5,10 @@ const helg = require('./');
 const wednesday = new Date(1414580400000);
 const saturday = new Date(1414839600000);
 const friday = new Date(1416600000000);
+const friday2 = new Date(1474020949000); // 09/16/2016 @ 10:15am (UTC)
```

Comment:
Not sure about this naming. What about `fridayBeforeNoon`?

Reply:
Fixed